sentry.client.config.js

```diff
@@ -0,0 +1,22 @@
+import { sentryDefaultConfig } from './sentry.default.config';
+import * as Sentry from '@sentry/nextjs';
+
+if (process.env.NEXT_PUBLIC_SENTRY_DSN) {
+  Sentry.init({
+    ...sentryDefaultConfig,
+    integrations: [
+      new Sentry.Integrations.Breadcrumbs({
+        console: false,
+        dom: false,
+        fetch: false,
+        history: false,
+        sentry: false,
+        xhr: false,
+      }),
+      new Sentry.Integrations.Dedupe(),
+      new Sentry.Integrations.FunctionToString(),
+      new Sentry.Integrations.GlobalHandlers(),
+      new Sentry.Integrations.HttpContext(),
+    ],
```

Comment:
After almost 2 hours of fighting with Sentry, I found this was the combination of integrations that gets use useful information without leaking users' configs (e.g. chain config with key-ed RPC url)